matchzoo/preprocessors/bert_preprocessor.py

```diff
@@ -0,0 +1,141 @@
+        """
+        Fit pre-processing context for transformation.
+
+        :param verbose: Verbosity.
+        :param data_pack: Data_pack to be preprocessed.
+        :return: class:`BertPreprocessor` instance.
+        """
+        data_pack = data_pack.apply_on_text(chain_transform(self._units),
+                                            verbose=verbose)
+        fitted_filter_unit = build_unit_from_data_pack(self._filter_unit,
+                                                       data_pack,
+                                                       flatten=False,
+                                                       mode='right',
+                                                       verbose=verbose)
+        self._context['filter_unit'] = fitted_filter_unit
+        self._context['vocab_unit'] = self._vocab_unit
+        vocab_size = len(self._vocab_unit.state['term_index'])
+        self._context['vocab_size'] = vocab_size
+        self._context['embedding_input_dim'] = vocab_size
+        self._context['input_shapes'] = [(self._fixed_length_left,),
+                                         (self._fixed_length_right,)]
+        return self
+
+    def transform(self, data_pack: DataPack, verbose: int = 1) -> DataPack:
+        """
+        Apply transformation on data, create fixed length representation.
+
+        :param data_pack: Inputs to be preprocessed.
+        :param verbose: Verbosity.
+
+        :return: Transformed data as :class:`DataPack` object.
+        """
+        data_pack = data_pack.copy()
+        data_pack.apply_on_text(chain_transform(self._units), inplace=True,
+                                verbose=verbose)
+
+        data_pack.apply_on_text(self._context['filter_unit'].transform,
+                                mode='right', inplace=True, verbose=verbose)
+        data_pack.apply_on_text(self._context['vocab_unit'].transform,
+                                mode='both', inplace=True, verbose=verbose)
+        data_pack.append_text_length(inplace=True, verbose=verbose)
+        data_pack.apply_on_text(self._left_fixedlength_unit.transform,
+                                mode='left', inplace=True, verbose=verbose)
+        data_pack.apply_on_text(self._right_fixedlength_unit.transform,
+                                mode='right', inplace=True, verbose=verbose)
+
+        max_len_left = self._fixed_length_left
+        max_len_right = self._fixed_length_right
+
+        data_pack.left['length_left'] = \
+            data_pack.left['length_left'].apply(
+                lambda val: min(val, max_len_left))
+
+        data_pack.right['length_right'] = \
+            data_pack.right['length_right'].apply(
+                lambda val: min(val, max_len_right))
+        return data_pack
+
+    @classmethod
+    def _default_units(cls) -> list:
+        """Prepare needed process units."""
+        return [
+            units.BertClean(),
+            units.BasicTokenize()

```

Comment:
you set `chinese_version` is `True` in `__init__`, why not use `ChineseTokenize` in default units?

Reply:
set the ``chinese_version`` is ``False`` for users'  different demands.